data/wanderer/wanderer news.txt

```diff
@@ -9,7 +9,43 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+news "wanderer civilians, no dialog 1, pre language"
+	to show
+		not "language: Wanderer"
```

Comment:
Sorry, my comment was not fully clear. What I meant is that we have a set of messages that is "always shown" and that don't refer to the Wanderers by name, and another set of messages that is "only shown after learning their name" and that does refer to the Wanderers by name.